### DISCUSSION TITLE
Allow sharing a server with a team

### DISCUSSION THREAD

**Discussion by dogboy21** [ServerListPlus/issues#21] [Ideas]
It would be great to have some sort of team functionality, so additional team members can be added to a server with editing permissions.

**Comment by Brend-Smits** [comment]
Thanks for the suggestion Dogboy21, I believe this functionality would be a great addition. 
Would you like to be able to configure what each user has permission to do? 
Take the following scenario's as an example:
1. User 'A' has permission to view/edit a server, but no permission to update the banner.
2. User 'B' has permission to view/edit a server, but no permission to delete a server.
3. User 'C' has permission to edit the team of a specific server.

Of course, as a server owner, you always have these permissions.

**Comment by dogboy21** [comment]
I think that additional team members should never have the permission to delete a server, only the owner should have this permission.
At least for our usecase, we don't need a permission system urgently, but I think in general it would be good to have at least 2 different permission groups:

- Editor: ability to edit stuff like the summary, description and banner
- Configurator: ability to edit the main server configuration like the server's address or the Votifier settings

**Reply by Brend-Smits** [reply]
That makes sense, thanks for the follow-up. I will see what I can do. 

If others have more suggestions regarding this feature, please feel free to comment.
